### PR TITLE
ENH: longer cache times etc

### PR DESCRIPTION
**Pull request by eagerterrier**
longer cache times for errors on topics + manifest/sw errors

### CODE REVIEW

File: src/server/index.jsx [hunk 1]
```diff
@@ -123,7 +123,10 @@ server
     async ({ params }, res) => {
       const { service } = params;
       const manifestPath = `${__dirname}/public/${service}/manifest.json`;
-      res.set('Cache-Control', 'public, max-age=604800');
+      res.set(
+        'Cache-Control',
+        'public, stale-if-error=1209600, stale-while-revalidate=1209600, max-age=604800',
```
Comment (andrewscfc) [inline src/server/index.jsx:128]: are you thinking by specifying all the directives they are more likely to make their way through to the user?

File: src/server/index.jsx [hunk 2]
```diff
@@ -142,12 +145,12 @@ const injectDefaultCacheHeader = (req, res, next) => {
   const defaultMaxAge = getDefaultMaxAge(req);
   const maxAge =
     req.originalUrl.indexOf('/topics/') !== -1
-      ? defaultMaxAge * 2
+      ? defaultMaxAge * 4
```
Comment (andrewscfc) [inline src/server/index.jsx]: Be good to monitor this when it goes out, we should see a material drop in topic requests I would hope! Fwiw I'd imagine up to 10 minutes might be okay for topics and given they are low traffic that might make the biggest difference?

File: src/server/index.jsx [hunk 3]
```diff
@@ -142,12 +145,12 @@ const injectDefaultCacheHeader = (req, res, next) => {
...
       : defaultMaxAge;
   res.set(
     'Cache-Control',
     `public, stale-if-error=${
-      maxAge * 4
+      maxAge * 10
```
Comment (andrewscfc) [inline src/server/index.jsx]: Isn't this meaningless: `stale-while-revalidate=${maxAge}, max-age=${maxAge}` ? A `stale-while-revalidate` and `max-age` being the same doesn't allow a window to serve stale while repopulating the cache in the background. As above, for topics it's a no brainer this should be 10 minutes for `stale-while-revalidate`. We should probably apply a lesser value for other assets as well?

File: src/server/index.jsx (hunk 1 above)
Comment (eagerterrier) [reply src/server/index.jsx:128]: there is a report where `stale-*` headers that are missing are logged. Trying to catch.

File: src/server/index.jsx (hunk 2 above)
Comment (eagerterrier) [reply src/server/index.jsx]: I'm nudging it up each time to see what the sweet spot is

**Pull request merged**